hw2/HomeWork2.java

```diff
@@ -0,0 +1,52 @@
+import java.util.Arrays;
+
+public class HomeWork2 {
+
+    public static void main(String[] args) {
+
+        int[] randomValuesArray = new int[10];
+        int minValue = randomValuesArray[0];
+        int maxValue = randomValuesArray[0];
+        int averageValue = 0;
+
+        for (int i = 0; i < randomValuesArray.length; i++) {
+            randomValuesArray[i] = (int) (Math.random() * 20);
+        }
+
+        System.out.println("Array with random values: " + Arrays.toString(randomValuesArray));
+
+        for (int numbers : randomValuesArray) {
+            if (numbers < minValue) {
+                minValue = numbers;
+            }
+        }
+
+        for (int numbers : randomValuesArray) {
+            if (numbers > maxValue) {
+                maxValue = numbers;
+            }
+        }
+
+        for (int numbers : randomValuesArray) {
+            averageValue = numbers + averageValue;
+        }
```

Comment:
You use 3 loops to find maxValue, minValue and averageValue. You can use only one.